src/visualisation/panel.cc

```diff
@@ -229,6 +230,9 @@ void VisualisationPanel::refreshVisualisation() {
 void VisualisationPanel::initLayout() {
   initOptionsPanel();
 
+  QWidget* root_widget = new QWidget(this);
+  this->setCentralWidget(root_widget);
```

Comment:
That's because QMainWindow has a special built-in layout - QMainWindowLayout. QMainWindow::setCentralWidget sets main (central/"content") widget for this particular layout. QMainWindow is a base class for veles::ui::View because we plan to support dockable toolbars in the docks. And this particular feature is supported by QMainWindowLayout.